modules/kubernetes/pages/index.adoc

```diff
@@ -23,15 +23,15 @@ This allows you to manage a TigerGraph cluster in a more natural and intuitive w
 
 Previous versions of TigerGraph used a different implementation of Kubernetes. This is still functional, but no longer supported. For long-term support and enhancement, use the Operator instead.
```

Comment:
```suggestion
Previous versions of TigerGraph used a different implementation of Kubernetes.
The older versions may be still functional, but no longer supported. For long-term support and enhancement, use the Operator instead.
```
Ambiguous what is "this".
Also, because this changeover took place a while ago now, I'm not positive that the older versions still work with the newer database.

Reply:
Thank you. this was changed in the latest commit.